packages/bitcore-wallet-client/src/lib/credentials.ts

```diff
@@ -132,6 +134,21 @@ export class Credentials {
 
     return x;
   }
+
+  /* 
+   * creates an ERC20 wallet from a ETH wallet
+   */
+  getTokenCredentials(token: { name: string, symbol: string, address:string }) {
```

Comment:
`address:string` needs space between

Reply:
we are using 
```
> tslint --fix -c tslint.json 'src/**/*.ts'
```
and it does not force that :/